torchvision/datapoints/_torch_function_helpers.py

```diff
@@ -0,0 +1,16 @@
+import torch
+
+_TORCHFUNCTION_SUBCLASS = False
+
+
+def set_return_type(type="Tensor"):
```

Comment:
This is the only function that is publicly exposed. We can bikeshed on the name a bit later.
It can also be a context manager.